node/src/chain_spec.rs

```diff
@@ -156,6 +153,93 @@ pub fn local_testnet_config(id: ParaId) -> ChainSpec {
 	)
 }
 
+pub fn rococo_testnet_root() -> AccountId {
+	hex!("baa78c7154c7f82d6d377177e20bcab65d327eca0086513f9964f5a0f6bdad56").into()
+}
+
+pub fn rococo_testnet_authorities() -> Vec<AuraId> {
+	use sp_core::crypto::UncheckedInto;
+
+	// ./scripts/prepare-test-net.sh 2
+	let initial_authorities: Vec<(AccountId, AccountId, AuraId)> = vec![
+		(
+			// 5Dkp2xwztRtRkXdmLDJUm8wvdHWSApgYgJY3rB5brKZkWJPM
+			hex!["4ad2d854d1bc5bbfdf93b689cb1aa3986684126c2b16d86924019de6798b0971"].into(),
+			// 5DvnJs3jM7sJkgfpVDxwu9GNmpFyzJSn2jG9w1hxBBWwqi92
+			hex!["526d7d3d5357d20e4cd75d1a452bd5b4903caf160f55d6bd19807efbdf165319"].into(),
+			// 5GKFbTTgrVS4Vz1UWWHPqMZQNFWZtqo7H2KpCDyYhEL3aS26
+			hex!["bc09354c12c054c8f6b3da208485eacec4ac648bad348895273b37bab5a0937c"]
+				.unchecked_into(),
+		),
+		(
+			// 5Gnadix7NJH2K7Akr5hVKyXqtYqTsQJbXnKFFt274MsG8n7h
+			hex!["d0e0ece66fb861b82383e85326a2e179316021105492820ca544ea8743620b59"].into(),
+			// 5FADgcPNMtcLtrmd16rZk6HJfDWqtEDT4N3jMSqtFo8tJhRr
+			hex!["88e95527362f479ebf30502db2f7d88329e034f5d77aed585042c548fa93ae01"].into(),
+			// 5EPRJHm2GpABVWcwnAujcrhnrjFZyDGd5TwKFzkBoGgdRyv2
+			hex!["66be63b7bcbfb91040e5248e2d1ceb822cf219c57848c5924ffa3a1f8e67ba72"]
+				.unchecked_into(),
+		),
+	];
+	initial_authorities.into_iter().map(|(_, _, aura_id)| aura_id).collect()
+}
+
+pub fn rococo_testnet_config(id: ParaId) -> ChainSpec {
+	// Give your base currency a unit name and decimal places
+	let mut properties = sc_chain_spec::Properties::new();
+	properties.insert("tokenSymbol".into(), "CAN".into());
+	properties.insert("tokenDecimals".into(), 12.into());
+
+	ChainSpec::from_genesis(
+		// Name
+		"Canvas",
+		// ID
+		"canvas_rococo",
```

Comment:
```suggestion
		"canvas-rococo",
```

Consistency. Local and rococo use the same spec id. That is OK? I did not found any information regarding this field apart from that id is the "id".

Reply:
I've distinguished them now.

Regarding the protocol id I just checked
* https://github.com/substrate-developer-hub/substrate-parachain-template/blob/master/node/src/chain_spec.rs
* https://github.com/paritytech/cumulus/blob/master/polkadot-parachains/src/chain_spec.rs

A protocol id is never set anywhere there, I've removed it from our chain spec as well now.

Reply:
It is used on the networking layer. We should keep it.

Reply:
I've added it back in after talking to Ricardo why they are missing in the code I linked above (seems to be a mistake there).